test/test.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+require('mocha');
+require('should');
```

Comment:
> we don't use should module anymore in this test.

good catch. Instead of just removing that line, I'll just remove `should` from the other file, so we can remove the dep completely. we're just doing string comparisons, so not much of a convenience to use `should` anyway...